pkg/controllers/cluster/taint_manager.go

```diff
@@ -235,15 +238,15 @@ func (tc *NoExecuteTaintManager) syncClusterBindingEviction(key util.QueueKey) e
 	}
 
 	var purgeMode policyv1alpha1.PurgeMode
+	var preservedLabelState map[string]string
 	if needEviction {
```

Comment:
I seem to have discovered a bug; when tolerationTime == 0, the following operation should also be executed.

What do you think?